ajax/libs/balance-text/package.json

```diff
@@ -13,6 +13,8 @@
         "files": [
           "jquery.balancetext.js",
           "jquery.balancetext.min.js"
+          "balancetext.js",
+          "balancetext.min.js"
```

Comment:
It will be more flexible to use `"balancetext*"` instead of these rules. Thanks.